models/core/core_models.yml

```diff
@@ -1,19 +1,555 @@
+        description: Date when the encounter started.
+      - name: encounter_end_date
+        description: Date when the encounter ended.
+      - name: admit_source_code
+        description: Admission source code for the encounter.
+        meta:
+          terminology: /terminology/administrative-codes/admit-source
+      - name: admit_source_description
+        description: Description of the admit_source_code for the encounter.
+      - name: admit_type_code
+        description: Admission type code for the encounter.
+        meta:
+          terminology: /terminology/administrative-codes/admit-type
+      - name: admit_type_description
+        description: Description of the admit_type_code for the encounter.
+      - name: discharge_disposition_code
+        description: Discharge disposition code for the encounter.
+        meta:
+          terminology: /terminology/administrative-codes/discharge-disposition
+      - name: discharge_disposition_description
+        description: Description of the discharge_disposition_code for the encounter.
+      - name: rendering_npi
+        description: Rendering NPI for the encounter (typically represents the physician or entity providing services).
+        meta:
+          terminology: /terminology/providers/provider
+      - name: billing_npi
+        description: Billing NPI for the encounter (typically represents organization billing the claim).
+        meta:
+          terminology: /terminology/providers/provider
+      - name: facility_npi
+        description: Facility NPI for the encounter (typically represents the facility where services were performed).
+        meta:
+          terminology: /terminology/providers/provider
+      - name: ms_drg_code
+        description: MS-DRG code for the encounter.
+        meta:
+          terminology: /terminology/conditions-and-procedures/ms-drg
+      - name: ms_drg_description
+        description: Description of the ms_drg_code.
+      - name: apr_drg_code
+        description: APR-DRG code for the encounter.
+        meta:
+          terminology: /terminology/conditions-and-procedures/ms-drg
+      - name: apr_drg_description
+        description: Description of the APR-DRG code.
+      - name: paid_date
+        description: The pay date for the encounter.
+      - name: paid_amount
+        description: The total amount paid for the encounter by the insurer.
+      - name: allowed_amount
+        description: The total amount allowed (includes amount paid by the insurer and patient) on the encounter.
+      - name: charge_amount
+        description: The total amount charged on the encounter by the provider.
+      - name: total_cost_amount
+        description: The total amount paid on the encounter by different parties.
+      - name: data_source
+        description: User-configured field that indicates the data source (e.g. typically named after the payer and state "BCBS Tennessee").
+
+
+  - name: core__lab_result
```

Comment:
I think we need to add the column names/descriptions for all of the new models. I believe the data dictionaries on the KB rely on this.